syntaxes/i3.tmLanguage

```diff
@@ -2282,9 +2282,9 @@
       </dict>
       <dict>
         <key>name</key>
-        <string>border</string>
+        <string>border_style</string>
         <key>match</key>
-        <string>(border)\s*(none|toggle|(\d+)\s*pixel|(?:pixel|normal)\s*(\d+))</string>
+        <string>(border)\s*(none|normal(\s*\d+)?|toggle(\s*\d+)?|pixel(\s*\d+)?)</string>
```

Comment:
Since the documentation states

> Note that "pixel" refers to logical pixel. On HiDPI displays, a logical pixel is represented by multiple physical pixels, so pixel 1 might not necessarily translate into a single pixel row wide border.

I believe it would be a good idea to highlight it! Thanks for pointing it out.
I'm committing this proposal.